exclusion.dic

```diff
@@ -34,3 +34,6 @@
 ﻿disksandvolumes
 ﻿Quickstart
 ﻿awaitable

```

Comment:
Oh nice, didn't realize we had this. I'll have to remember to add to this in the future.

Reply:
If you hover over a word VS is complaining about there's an option to "ignore" which adds it to this file. (Unless it decides to be difficult and create a local dictionary for your VS installation, but it only did that to me once)